build.gradle

```diff
@@ -21,7 +22,30 @@ buildscript {
 allprojects {
     repositories {
         google()
+        mavenCentral()
         jcenter()
         maven { url 'https://maven.fabric.io/public' }
     }
-}
\ No newline at end of file
+}
+
+ext {
```

Comment:
Same thing in there

Reply:
We'll do it later! Unless you want to do it on another branch...